exercicios/projeto/Client/Client.js

```diff
@@ -0,0 +1,35 @@
+import Account from "../Account/Account.js"
+
+class Client {
+    name;
+    #cpf;
+    #account;
+    #income;
+
+    registerClient(name, cpf, account, income) {
+        if (account instanceof Account) {
+            this.name = name;
+            this.#cpf = cpf;
+            this.#account = account;
+            this.#income = income;
+
+            return "Cliente cadastrado";
+        } else {
+            throw new Error("Erro no cadastro, dados inválidos");
+        }
+    }
+
+    getIncome() {
```

Comment:
Muito bom você ter implementado esse método pra identificação do tipo de cliente, porém como nessa entrega você não o utiliza ele se faz desnecessário. Porém em um segundo momento quando e se você for fazer uma refatoração  para atender ao projeto todo ele será muito importante.